modules/geo-layers/package.json

```diff
@@ -40,7 +40,8 @@
     "@math.gl/culling": "3.6.0-alpha.5",
     "@math.gl/web-mercator": "3.6.0-alpha.5",
     "h3-js": "^3.6.0",
-    "long": "^3.2.0"
+    "long": "^3.2.0",
+    "ngeohash": "^0.6.3"
```

Comment:
yeah, I wasn't sure about this and decided to just include the library.  Since this approach was done for some other cases, I swapped ngeohash out and put in just the relevant code instead and made appropriate updates in other places to remove the library.